Source/WebCore/bindings/js/ScriptModuleLoader.cpp

```diff
@@ -195,12 +195,12 @@ JSC::JSInternalPromise* ScriptModuleLoader::fetch(JSC::JSGlobalObject* jsGlobalO
         return jsPromise;
     }
 
-    RefPtr<ModuleFetchParameters> topLevelFetchParameters;
+    RefPtr<JSC::ScriptFetchParameters> assertions;
     if (auto* scriptFetchParameters = JSC::jsDynamicCast<JSC::JSScriptFetchParameters*>(parameters))
-        topLevelFetchParameters = static_cast<ModuleFetchParameters*>(&scriptFetchParameters->parameters());
+        assertions = &scriptFetchParameters->parameters();
```

Comment:
Are these assertions or parameters? We should probably stick to one terminology.

Reply:
We should name it as `parameters`, since we can have non-assertion data too. (And dynamic `import` is designed actually to take non assertions in the future).